modules/release-notes/con-relnotes-fixed-issues.adoc

```diff
@@ -220,6 +220,35 @@ With the release of the {product-short} 1.2.1 Helm chart, this is fixed.
 
 == Fixed security issues
 
+== Fixed issues in {product} 1.2.6
```

Comment:
move this into a `snip-fixed-security-issues-in-*.adoc` file using the provided bash script. 

Also run the python script to generate any new RN/KI content

Reply:
Yes, running ` ./single-source-fixed-security-issues.sh 1.2.6` created the adoc file: https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/755/commits/4684cf3697c79d315f5ee93cbb3cc396f58eb1cc